src/main/java/net/minecraftforge/event/ForgeEventFactory.java

```diff
@@ -198,6 +201,24 @@ public static Result canEntityDespawn(EntityLiving entity)
         return event.getResult();
     }
 
+    public static int getItemBurnTime(@Nonnull ItemStack itemStack)
+    {
+        Item item = itemStack.getItem();
+        int burnTime = item.getItemBurnTime(itemStack);
```

Comment:
Where is the patch for this method in `Item.java`? I don't see a change in `Item.java.patch`.